examples/launch_pasha_nasbench201.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
OK, I prefer this to having code as part of the tutorial. You can link to this example from the tutorial, but code in examples is going to be kept running.